src/test/osd/osd-scrub-snaps.sh

```diff
@@ -151,16 +151,14 @@ function TEST_scrub_snaps() {
     wait_for_clean || return 1
 
     sleep 5
-    ceph pg scrub ${poolid}.0
-    timeout 30 ceph -w
+    pg_scrub "${poolid}.0"
+    timeout 1 ceph -w
```

Comment:
**timeout 30** seems appropriate here: on a slow machine it may take a few seconds for **ceph -w** to display the information that we need to debug problems (i.e. it will show the lines that are supposed to be matched later in the script). Because you added pg_scrub above, the sleep 30 is no longer necessary as a time to wait for the scrub to complete, which is good :-)